src/main/java/com/iridium/iridiumskyblock/database/Island.java

```diff
@@ -132,11 +134,21 @@ public void setColor(Color color) {
     @Override
     public @NotNull String getName() {
         if (super.getName() != null) return super.getName();
-        String ownerName = IridiumSkyblock.getInstance().getTeamManager().getTeamMembers(this).stream()
-                .filter(user -> user.getUserRank() == Rank.OWNER.getId())
-                .findFirst()
+        String ownerName = getOwner()
                 .map(User::getName)
                 .orElse("N/A");
         return IridiumSkyblock.getInstance().getConfiguration().defaultIslandName.replace("%owner%", ownerName);
     }
+
+    public List<User> getMembers()
+    {
+        return IridiumSkyblock.getInstance().getTeamManager().getTeamMembers(this);
+    }
+
+    public Optional<User> getOwner()
+    {
+        return getMembers().stream()
+                .filter(user -> user.getUserRank() == Rank.OWNER.getId())
+                .findFirst();
+    }
```

Comment:
Really don't like how inconsistent this is, why suddenly use the C# bracket style